src/index.html

```diff
@@ -5,8 +5,434 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="logo-link">
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="menu">
+          <li class="menu__item">
+            <a href="#" class="menu__item-link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__item-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              Photo
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="products">
+        <div class="card card--margin" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt=" photo of APPLE A1419 iMac 27 Retina 5K Monoblock"
+            class="card__photo"
+          >
+
+          <h2 class="card__info">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="rating">
```

Comment:
You shouldn't have additional BEM blocks besides header, nav, card, stars (ok let it be with main and products)
```suggestion
          <div class="card__rating">
```